api/src/controllers/theme.js

```diff
@@ -5,10 +5,10 @@
 const Theme = require('../models/editor/theme.model');
 
 /**
- * 
- * @param {*} req 
- * @param {*} res 
- * @returns 
+ *
+ * @param {*} req
+ * @param {*} res
+ * @returns
```

Comment:
This probably needs to be filled in later.

Reply:
Added a story to winter's board as a stopgap since I believe one of you worked on theme:
https://trello.com/b/5qazh84D/team-winter